File: testdata/bench/swift/BenchmarkTest00002.swift
import Vapor
import SQLite3

func handler002(_ req: Request) throws -> String {
    let name: String = try req.query.get(at: "name")
    let query = "SELECT * FROM users WHERE name = '" + name + "'"
    var db: OpaquePointer?
    sqlite3_open(":memory:", &db)
    sqlite3_exec(db, query, nil, nil, nil)
    return "ok"
}

File: testdata/bench/go/BenchmarkTest00002.go
package bench

import (
	"database/sql"
	"fmt"
	"net/http"
)

// SQL injection via fmt.Sprintf
func Handler00002(w http.ResponseWriter, r *http.Request) {
	param := r.FormValue("name")
	query := fmt.Sprintf("SELECT * FROM users WHERE name = '%s'", param)
	db, _ := sql.Open("sqlite3", ":memory:")
	db.Exec(query)
}
